applications/Unity.GrantManager/src/Unity.GrantManager.Web/Views/Shared/Components/CommentsWidget/Default.cshtml

```diff
@@ -48,7 +48,15 @@
                             @comment.Comment
                         </div>
                     </div>
-                    <div class="commented-time mt-3">@comment.CreationTime.ToString("yyyy-MM-dd h:mm tt")</div>
+                    @if (@comment.LastModificationTime != null)
+                    {
+                        <div class="commented-time mt-3">Modified: @comment.LastModificationTime?.ToString("yyyy-MM-dd h:mm tt")</div>
+                    }
+                    else
+                    {
+                        <div class="commented-time mt-3">@comment.CreationTime.ToString("yyyy-MM-dd h:mm tt")</div>
```

Comment:
Should we not add the (L:created) in front of this timestamp

Reply:
@AndreGAot  Updated